rest-api/dao/participant_summary_dao.py

```diff
@@ -235,6 +235,7 @@ def update_from_biobank_stored_samples(self, participant_id=None):
       samples_to_isolate_dna = (
           CASE WHEN EXISTS(SELECT * FROM biobank_stored_sample
                            WHERE biobank_stored_sample.biobank_id = participant_summary.biobank_id
+                           AND biobank_stored_sample.confirmed IS NOT NULL
```

Comment:
This is only going to affect samples_to_isolate_dna. You also need to update the SQL for num_baseline_samples_arrived and in _get_sample_sql_and_params in here, as well as the reconciliation report:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/offline/biobank_samples_pipeline.py#L302

Basically grep for biobank_stored_sample and make sure that all the places we reference it are filtering out samples without confirmed dates.

Reply:
done